Tests/Foundation/UndoManagerMonitorTests.swift

```diff
@@ -0,0 +1,247 @@
+//
+//  UndoManagerMonitorTests.swift
+//  XestiMonitorsTests
+//
+//  Created by Rose Maina on 2018-04-30.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+import UIKit
```

Comment:
@rosemaina 
UIKit is not needed here and is actually causing the build to fail on macOS.